JiraPS/Public/Find-JiraFilter.ps1

```diff
@@ -0,0 +1,89 @@
+function Find-JiraFilter {
+    # .ExternalHelp ..\JiraPS-help.xml
+    [CmdletBinding( SupportsPaging )]
+    param(
+        [Parameter(ValueFromPipeline,ValueFromPipelineByPropertyName)]
+        [string[]]$Name,
+
+        [Parameter(ValueFromPipelineByPropertyName)]
+        [string]$AccountId,
+
+        [Parameter(ValueFromPipelineByPropertyName)]
+        [string]$GroupName,
+
+        [Parameter(ValueFromPipelineByPropertyName)]
+        [Object]$Project,
```

Comment:
Please add this temporary validation of `JiraPS.Project`

https://github.com/AtlassianPS/JiraPS/blob/9786aa03798de8a75b4ebdb897b5729472a88f13/JiraPS/Public/Get-JiraComponent.ps1#L7-L27

Reply:
Adding validation scripts for both Project and Owner.  Are they covered by unit tests, or explicitly excluded from code coverage enforcement, elsewhere?